pyhf/paramsets.py

```diff
@@ -53,11 +52,13 @@ def reduce_paramset_requirements(paramset_requirements):
 
     for param_name in list(paramset_requirements.keys()):
         params = paramset_requirements[param_name]
```

Comment:
I realize I'm to blame for this, but this should probably be `reqs` or something similar